modules/db-writer/src/main/resources/analytics.sql

```diff
@@ -21,13 +21,16 @@ create table if not exists executed_deposit (
     stake_pkh Text,
     collateral_ada BIGINT not null,
     order_input_id Text not null,
-    user_output_id Text not null,
-    pool_input_Id Text not null,
-    pool_output_Id Text not null,
-    timestamp BIGINT not null
+    user_output_id Text,
+    pool_input_Id Text,
+    pool_output_Id Text,
+    redeem_output_Id Text,
+    creation_timestamp BIGINT,
+    execution_timestamp BIGINT,
```

Comment:
We need refund ts and transactions ids

Reply:
Execution timestamp row can be used in several ways:
- To store execution tx timestamp
- To store refund tx timestamp
But, looks like naming is not too clear